flink-runtime/src/main/java/org/apache/flink/runtime/leaderelection/ZooKeeperLeaderElectionDriver.java

```diff
@@ -128,20 +128,20 @@ public boolean hasLeadership() {
     }
 
     @Override
-    public void publishLeaderInformation(String contenderID, LeaderInformation leaderInformation) {
+    public void publishLeaderInformation(String componentId, LeaderInformation leaderInformation) {
         Preconditions.checkState(running.get());
 
         if (!leaderLatch.hasLeadership()) {
             return;
         }
 
         final String connectionInformationPath =
-                ZooKeeperUtils.generateConnectionInformationPath(contenderID);
+                ZooKeeperUtils.generateConnectionInformationPath(componentId);
 
         LOG.debug(
-                "Write leader information {} for {} to {}.",
+                "Write leader information {} for component '{}' to {}.",
```

Comment:
some log message use "component" while others use "component id". I'd prefer "component" because it reads much more naturally.